### DISCUSSION TITLE
tvOS 15.4 Issue?

### DISCUSSION THREAD

**Discussion by jtochterman** [dgreif/ring#904] [Q&A]
Updated all my devices last night and all my HomeKit + HomeBridge stuff seems to be working great on all platforms (iOS, iPadOS, MacOS - Catalina) except tvOS 15.4.  You go to look at the camera, the thumbnail disappears and the status says "No Response." I do not even see an attempt to open the stream in the HB logs. Very strange behavior. I'm assuming the issue is on tvOS devices and not HB as I have 3 devices and they are all doing the same thing. Any ideas?

![image](https://user-images.githubusercontent.com/4764338/158481421-bed1f119-4dbf-4256-aa0d-6cf1b74371ce.png)

![image](https://user-images.githubusercontent.com/4764338/158481433-04662086-1b98-4c6e-8d3e-384b22746a3b.png)

**Comment by dgreif** [comment]
If tvOS is the only instance where it isn't working then this is an issue in HomeKit and not likely something that I can fix in the plugin unfortunately

**Reply by jtochterman** [reply]
That's what I was thinking as well. When I look at the HomeBridge logs, I don't even seeing it making an attempt to connect. All the other devices do, just the Apple TV's. I was just hoping others had seen this behavior to validate that I wasn't crazy... at least on this. LOL

**Reply by dgreif** [reply]
I've seen one or two other similar reports 😄
